hadoop-hdds/docs/content/feature/OM-HA.md

```diff
@@ -56,7 +56,7 @@ This logical name is called `serviceId` and can be configured in the `ozone-site
  ```
```

Comment:
This code block looks different in the [rendered doc](https://ozone.apache.org/docs/1.2.1/feature/om-ha.html#configuration).  The problem is not introduced by this patch, but since you are already making a change here, can you please fix it?

```suggestion
```XML
```

Reply:
resolved

Reply:
@adoroszlai Thank you for suggestion, will raise another PR